### ISSUE TITLE
Incrementing/decrementing the year, prior to 1883, causes 1 year 8 minutes 4 seconds to change(instead of 1 year exactly) 

### ISSUE THREAD

**Issue by BradleyPelton** [Stellarium/stellarium-web-engine#47]
TLDR: watch https://gyazo.com/d24049629e9a43c3b52b81e03ac34abf
SUMMARY: When changing the year prior to 1883, an abnormal amount of time changes. Instead of the normal 1 year, 1 year 8 minutes 4 seconds changes on the DATE_TIMESTAMP clock. Only seems to occur prior to 1883 
PRIORITY: Low 
NOTES: The increment/decrement buttons can be unresponsive after several clicks. I assume its probably due to my poor laptop, but it might be worth investigating. You can see it in the clip above.
### Expected Behavior
The time delta should be exactly 1 year
### Actual Behavior
Its 1year 8 minutes 4 seconds
Steps to reproduce
1.) Change the year to 1883
2.) decrement the year to 1882
3.) The hour and minute clock will incorrectly be altered.

### Gif of execution
https://gyazo.com/8a6adda8bff1d7c228436aab63e67d18

**Comment by gzotti** [comment]
Guess: 1883 is the first year where timezones are used? Before that, you may jump to local mean solar time which depends on your longitude.
